### PR TITLE
Remove engine usage from events

### PR DESCRIPTION
**Pull request by LK4D4**
ping @tiborvass

### CODE REVIEW

**Comment by LK4D4** [comment]
ping @tiborvass @crosbymichael

**Comment by LK4D4** [comment]
@tiborvass I renamed to `Log`

**Comment by tiborvass** [comment]
LGTM

**Comment by LK4D4** [comment]
ping @icecrime

**Comment by LK4D4** [comment]
ping @jfrazelle

File: daemon/events/events.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	e.mu.Unlock()
+	return current, l
+}
+
+// Evict evicts listener from pubsub
+func (e *Events) Evict(l chan interface{}) {
+	e.pub.Evict(l)
+}
+
+// Log broadcasts event to listeners. Each listener has 100 millisecond for
```
Comment (icecrime) [inline daemon/events/events.go]: I think that timeout logic is gone in the `Log` function (I'm not sure why that changed however).

**Comment by icecrime** [comment]
LGTM

**Pull request merged**